### PR TITLE
Integrate with Travis CI

### PR DESCRIPTION
**Pull request by franciozzy**
This adds a simple .travis.yml file for integration with Travis CI. It
tries to build muser.git on a Ubuntu Bionic environment using the Linux
kernel 5.2.21.

Signed-off-by: Felipe Franciosi <felipe@nutanix.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: c
+dist: bionic
+compiler: gcc
+before_install:
+  - src=$(pwd)
+  - mkdir /tmp/linux
```
Comment (tmakatos) [inline .travis.yml]: It would bee better to use `/tmp/linux` from a variable.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - cd /tmp/linux
+  - git init
+  - git remote add origin git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
+  - git fetch --depth 1 origin e91ef5bcdeda8956eb9f1972ed90198b698dca0f # 5.2.21
```
Comment (tmakatos) [inline .travis.yml]: It's cleaner to use the tag `v5.2.21` than the SHA.

File: lib/libmuser.c [hunk 3]
```diff
@@ -45,6 +45,7 @@
 #include <stddef.h>
 #include <sys/mman.h>
 #include <stdarg.h>
+#include <linux/vfio.h>
```
Comment (tmakatos) [inline lib/libmuser.c]: Why is this needed?

File: .travis.yml (hunk 1 above)
Comment (franciozzy) [reply .travis.yml]: Sure. I'll also fix the other variable names.

File: .travis.yml (hunk 2 above)
Comment (franciozzy) [reply .travis.yml]: I thought only SHA1 worked (from reading some online doc) and never tried the tag. Tried it now separately and it works, so I'll do that.

File: lib/libmuser.c (hunk 3 above)
Comment (franciozzy) [reply lib/libmuser.c]: Actually that slipped here, should probably have been a separate commit, but I don't think it's worth separating because only Travis breaks. Travis couldn't build libmuser without it, presumably because `libmuser.c` uses VFIO pound-defined constants that happen to be available via some other include or source file in the same compilation unit. I didn't spend absolutely any time thinking about why it didn't work, I just added it here.
